pkg/server/io/container_io.go

```diff
@@ -225,21 +213,9 @@ func (c *ContainerIO) Attach(opts AttachOptions) error {
 	return nil
 }
 
-// Cancel cancels container io.
-func (c *ContainerIO) Cancel() {
-	c.closer.Cancel()
-}
-
-// Wait waits container io to finish.
+// Wait for IO streams to end
 func (c *ContainerIO) Wait() {
-	c.closer.Wait()
-}
-
-// Close closes all FIFOs.
-func (c *ContainerIO) Close() error {
-	c.closer.Close()
-	if c.fifos != nil {
-		return os.RemoveAll(c.fifos.Dir)
+	if c.wg != nil {
+		c.wg.Wait()
```

Comment:
Hm, this doesn't seem correct to me. The `DirectIO.Wait` function seems never called.

Reply:
`DirectIO.Wait()` is a no-op, so no reason to call it.